packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx

```diff
@@ -1,19 +1,87 @@
-import React from 'react';
+import {
+  EmptyStateComingSoon,
+  EncounterList,
+  EncounterListColumn,
+  PatientChartProps,
+  SummaryCard,
+  SummaryCardColumn,
+  getObsFromEncounter,
+} from '@ohri/openmrs-esm-ohri-commons-lib';
+import { useConfig } from '@openmrs/esm-framework';
+import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { EmptyStateComingSoon } from '@ohri/openmrs-esm-ohri-commons-lib';
+import { moduleName } from '../../..';
 
-interface TptPatientSummaryProps {
-  patientUuid: string;
-}
-
-const TptPatientSummary: React.FC<TptPatientSummaryProps> = ({ patientUuid }) => {
+const TptPatientSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const config = useConfig();
```

Comment:
A nice trick is you can do is destructure this to shorten the names like so 

```
const { obsConcepts, encounterTypes } = useConfig();
```

Then down you can just 
```
encounterTypes.tptCaseEnrollment

OR

obsConcepts.tptEnrollmentDate
```

Reply:
Great! Thanks